### PR TITLE
Add XCM SDK docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds the new XCM SDK docs! 🎉  These are versioned docs, so the docs for v0-v2 are all available in one central place.

For mkdocs, versioning is made possible through [mike](https://github.com/jimporter/mike), which works by creating a new Git commit on the `gh-pages` branch every time a new version of the docs is deployed using `mike deploy` commands (see `mkdocs/README.md`). When deploying a particular version, previously-deployed docs for that version are overwritten, but docs for other versions remain untouched. So we'll need to make sure that the `gh-pages` branch is always open and never manually modified.

🔁 The current workflow can be summarized as:

- Make changes and open a PR to be merged into the `main` branch
- Run the `mike deploy` commands locally, which will push up changes automatically to the `gh-pages` branch of your fork
- Open a PR to merge your fork changes into the `gh-pages` branch

There may be a better way to automate or handle this, but at least with this flow, we'll be able to dictate whether changes result in updating the latest version or publishing a new version.

✅ For initial setup:

1. Go to the **settings** page of the repo
2. Under **code and automation**, click **Pages**
3. For the **Build and deploy** settings, the **source** should be set to **Deploy from a branch** and the **Branch** should be set to **`gh-pages`**

Anytime we merge new version changes to the `gh-pages` branch, it will automatically deploy and publish those changes to https://moonbeam-foundation.github.io/xcm-sdk/ 🚀 

👀  To see how it currently looks, you can checkout my test page: [https://eshaben.github.io/xcm-sdk-mkdocs/latest/](https://eshaben.github.io/xcm-sdk-mkdocs/latest/)

### Checklist

- [ ] If this requires a documentation change, I have created a PR in [moonbeam-docs] (https://github.com/moonbeam-foundation/moonbeam-docs) repository.
- [x] If this requires it, I have updated the Readme
- [ ] If necessary, I have updated the examples
- [ ] I have verified if I need to create/update unit tests
- [ ] I have verified if I need to create/update acceptance tests
- [ ] If necessary, I have run acceptance tests on this branch in CI

### CODE REVIEW

**Review by ekenigs — COMMENTED** [review]
I see in some places:
```typescript
const sdkInstance = new Sdk();
const transferDataWithoutAccounts = sdkInstance
```
`Sdk()` is not a class so you can't call it with `new`.

**Review by ekenigs — CHANGES_REQUESTED** [review]
Hey Erin, good job for the PR, and sorry that we waited so long with it. I notice a few things.

File: packages/config/src/assets.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import { Asset } from '@moonbeam-network/xcm-types';
 
+export const erin = new Asset({
```
[the remark this answers is not in the document]
Comment (eshaben) [reply packages/config/src/assets.ts:3]: haha OOPS! nope 🙂

**Comment by eshaben** [comment]
thanks for the review @ekenigs! applied your feedback 🙂 

I'll need to go back through and redeploy v1,  but I'd like to do that after I get a review of the changes I just pushed, so I only have to redeploy once.

**Review by ekenigs — APPROVED** [review]

**Review by ekenigs — APPROVED** [review]

**Pull request merged**